src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -119,6 +120,18 @@ public void removeSession(Name name, Index sessionIndex) {
 
     }
 
+    /**
+     * Removes a {@code Session} with {@code sessionDate} from {@code sessionIndex} of {@code name}
```

Comment:
Hmmmm for me this description is quite confusing also. Maybe something like "Removes a single {@code Session} with {@code sessionDate} from the recurring session and splits up the remaining sessions into 2 separate recurring sessions. We identify the recurring sessions to perform this action using {@code sessionIndex} and {@code name}." would be better?